pkg/cmd/pulumi/auth/login.go

```diff
@@ -205,19 +208,53 @@ func NewLoginCmd() *cobra.Command {
 	return cmd
 }
 
-func validateCloudBackendType(typ string) error {
-	kind := strings.SplitN(typ, ":", 2)[0]
+// validateCloudBackendType validates the cloud backend type.
+// It returns true if the cloud backend type is http or https.
+// It returns false if the cloud backend type is azblob, gs, s3, or file.
+// It returns an error if the cloud backend type is not supported.
+func validateCloudBackendType(typ string) (bool, error) {
+	parseURL, err := url.Parse(typ)
+	if err != nil {
+		return false, fmt.Errorf("unknown backend cloudUrl format '%s' (supported Url formats are: "+
```

Comment:
this should error that the url is invalid and maybe include the error from url.parse